testers/rdf-fixtures/fixture-tables/assume-world-writable.ttl

```diff
@@ -3,105 +3,106 @@
-                            ] .
-
-<#public-cors-origin-unset-req> a http:RequestMessage ;
-                    http:method "GET" ;
-                    http:requestURI </public/verypublic/foobar.ttl> .
-
-<#public-cors-origin-unset-res> a http:ResponseMessage ;
-                    http:status 200 ;
-                    httph:access_control_allow_origin "*" .
-
-<#public-options> a test:AutomatedTest ;
-                test:purpose "Testing OPTIONS method"@en ;
-                rdfs:isDefinedBy <https://github.com/solid/solid-spec/blob/b941ff795acdedb7d7a24d40dabdfcce7efa9283/api-rest.md#discovering-server-capabilities---the-options-method>, <https://www.w3.org/TR/cors/#syntax> ;
-                test:test_script <http://example.org/httplist#http_req_res_list_unauthenticated> ;
-                test:params [
-                    test:requests ( <#public-options-req> ) ;
-                    test:responses ( <#public-options-res> )
-                            ] .
-
-<#public-options-req> a http:RequestMessage ;
-                    http:method "OPTIONS" ;
-                    http:requestURI </public/> .
-
-<#public-options-res> a http:ResponseMessage ;
-                    http:status 200 ; # TODO: For this, we need bag and possibly subset tests
-                    httph:accept_patch	"application/json", "application/sparql-update" ;
-                    httph:accept_post	"text/turtle", "application/ld+json" ;
-                    httph:access_control_allow_credentials	"true" ;
-                    httph:access_control_allow_methods	"OPTIONS", "HEAD", "GET", "PATCH", "POST", "PUT", "DELETE" ;
-                    httph:access_control_allow_origin	"*" ;
-                    httph:access_control_expose_headers	"User", "Triples", "Location", "Link", "Vary", "Last-Modified", "Content-Length" ;
-                    httph:allow	"OPTIONS", "HEAD", "GET", "PATCH", "POST", "PUT", "DELETE" .
-                                        
+
+
+:public_cors_origin_set a test:AutomatedTest ;
+    test:purpose "Testing CORS header when Origin is supplied by client"@en ;
+    rdfs:isDefinedBy <https://www.w3.org/TR/cors/#syntax>, <https://github.com/solid/solid-spec/blob/master/recommendations-server.md#cors---cross-origin-resource-sharing> ;
+    test:test_script <http://example.org/httplist#http_req_res_list> ;
+    test:params [
+        test:requests ( :public_cors_origin_set_req ) ;
+        test:responses ( :public_cors_origin_set_res )
+    ] .
+
+:public_cors_origin_set_req a http:RequestMessage ;
+    http:method "GET" ;
+    httph:origin <https://app.example> ;
+    http:requestURI </public/verypublic/foobar.ttl> .
+
+:public_cors_origin_set_res a http:ResponseMessage ;
+    http:status 200 ;
+    httph:access_control_allow_origin <https://app.example> .
+
+
+:public_cors_origin_unset a test:AutomatedTest ;
+    test:purpose "Testing CORS header when Origin is not supplied by client"@en ;
```

Comment:
Do you think we should write test descriptions as we do in Mocha.

Like "provides a CORS header when Origin is not supplied by the client"

And it would be printed as "It provides a CORS header when Origin is not supplied by the client"

Reply:
You mean, the test description? Yeah, that's certainly up for the grabs.

BTW; those tests needs to be rewritten, as they assume we start from world-writeable, and that's not an assumption we should start with.